packages/SwingSet/src/kernel/state/kernelKeeper.js

```diff
@@ -31,12 +31,17 @@ const enableKernelPromiseGC = true;
 // The schema is:
 //
 // vat.names = JSON([names..])
+// vat.dynamicIDs = JSON([vatIDs..])
 // vat.name.$NAME = $vatID = v$NN
 // vat.nextID = $NN
 // device.names = JSON([names..])
 // device.name.$NAME = $deviceID = d$NN
 // device.nextID = $NN
 
+// dynamic vats have these too:
+// v$NN.source = JSON({ bundle }) or JSON({ bundleName })
```

Comment:
I suspect overloading this is the tail that wagged to dog w.r.t the API complexity I complained about above.

Reply:
It did make it make the other pieces fall into that shape, yeah. My first pass had separate `v$NN.sourceBundle` and `v$NN.sourceBundleName`, with a comment that there should only ever be one, which is a sort of fussy invariant that I always want to avoid (better to have the rule be present-or-not than one-of-two).